cmd/statusd-cli/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+)
+
+const (
+	// StatusDHost is the default host to connect to.
```

Comment:
Minor but we usually refer to it `statusd` so `StatusdHost` would be more consistent.

Reply:
Done based on comment below.